src/SURFnet/SPRegistration/Janus/ArpMapper.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace SURFnet\SPRegistration\Janus;
+
+use AppBundle\Entity\Subscription;
+use AppBundle\Model\Attribute;
+use OpenConext\JanusClient\ArpAttributes;
+
+/**
+ * Class ArpMapper
+ * @package SURFnet\SPRegistration\Janus
+ */
+final class ArpMapper
+{
+    /**
+     * @param Subscription $request
+     * @return null|ArpAttributes
```

Comment:
Ah nee, dat is er uit gehaald. Een NULL ARP betekend geen ARP, m.a.w. alle attributen. Maar dat gebruikt het formulier niet.